### PR TITLE
Replace missing close tag

### PR DESCRIPTION
**Pull request by gemma-shay**
was accidentally deleted while switching to product name variables

### CODE REVIEW

File: v20.2/connect-to-the-database.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ toc: true
 
 <div class="filters filters-big clearfix">
   <button class="filter-button current">Local</button>
-  <a href="connect-to-the-database-cockroachcloud.html"><button class="filter-button"{{ site.data.products.dedicated }}
+  <a href="connect-to-the-database-cockroachcloud.html"><button class="filter-button">{{ site.data.products.dedicated }}
```
Comment (ianjevans) [inline v20.2/connect-to-the-database.md:9]: Needs a `</button></a>` too.
```suggestion
  <a href="connect-to-the-database-cockroachcloud.html"><button class="filter-button">{{ site.data.products.dedicated }}</button></a>
```

File: v21.1/connect-to-the-database.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ toc: true
 
 <div class="filters filters-big clearfix">
   <button class="filter-button current">Local</button>
-  <a href="connect-to-the-database-cockroachcloud.html"><button class="filter-button"{{ site.data.products.dedicated }}
+  <a href="connect-to-the-database-cockroachcloud.html"><button class="filter-button">{{ site.data.products.dedicated }}
```
Comment (ianjevans) [inline v21.1/connect-to-the-database.md:9]:
```suggestion
  <a href="connect-to-the-database-cockroachcloud.html"><button class="filter-button">{{ site.data.products.dedicated }}</button></a>
```

**Review by ianjevans — CHANGES_REQUESTED** [review]

File: v21.1/connect-to-the-database.md (hunk 2 above)
Comment (gemma-shay) [reply v21.1/connect-to-the-database.md]: whoops thank you! and sorry for all the pushes

File: v21.1/connect-to-the-database-cockroachcloud.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ toc: true
 
 <div class="filters filters-big clearfix">
   <a href="connect-to-the-database.html"><button class="filter-button">Local</button></a>
-  <button class="filter-button current">{{ site.data.products.serverless }}</button>
+  <button class="filter-button current">{{ site.data.products.db }}</button>
```
Comment (ianjevans) [inline v21.1/connect-to-the-database-cockroachcloud.md:9]:
```suggestion
  <button class="filter-button current">{{ site.data.products.db }}</button></a>
```

**Review by ianjevans — CHANGES_REQUESTED** [review]

File: v21.1/connect-to-the-database-cockroachcloud.md (hunk 3 above)
Comment (gemma-shay) [reply v21.1/connect-to-the-database-cockroachcloud.md:9]: I think the `filter-button current` ones are all just button elements not urls

**Review by ianjevans — APPROVED** [review]

File: v21.1/connect-to-the-database-cockroachcloud.md (hunk 3 above)
Comment (ianjevans) [reply v21.1/connect-to-the-database-cockroachcloud.md:9]: > I think the `filter-button current` ones are all just button elements not urls

Yup, duh.

**Pull request merged**